main.py

```diff
@@ -115,7 +115,7 @@ def buy():
                         amount = left
 
                     logger.info(
-                        f'starting buy place_order with : {announcement_coin=} | {globals.pairing=} | {volume=} | {amount=} x {price=} | side = buy | {status=}',
+                        f'starting buy place_order with : {announcement_coin} | {globals.pairing} | {volume} | {amount} x {price} | side = buy | {status}',
```

Comment:
I just found out that this is specific to python 3.8 fstrings and does not work in python 3.7:
https://stackoverflow.com/questions/66612440/is-fexpr-a-new-f-string-expression-see-equal-sign-before-training-brace

Which python version is officially supported and recommended for this bot?

Reply:
Also see Issue #114
'python3' probably symlinked to an older python version (<3.8) and therefore cause the syntax error

Reply:
I reverted these changes and upgraded the scripts and gh-actions to python versions 3.8+ because thats the version officially recommended here:

https://www.cryptomaton.org/2021/10/17/a-binance-and-gate-io-crypto-trading-bot-for-new-coin-announcements/

Reply:
Yes, that's correct, it's Python 3.8+